Civi/Core/Container.php

```diff
@@ -206,6 +206,12 @@ public function createContainer() {
       []
     ))->setPublic(TRUE);
 
+    $container->setDefinition('bundle.coreStyles', new Definition('CRM_Core_Resources_Bundle', ['coreStyles']))
+      ->setFactory('CRM_Core_Resources_Common::createStyleBundle');
+
```

Comment:
Do these need setPublic(TRUE) or is there something you need to configure on a site? I'm having trouble after updating a drupal 8 site and adding setPublic makes it work.